src/agentscope/studio/static/html-drag-components/tool-code.html

```diff
@@ -0,0 +1,17 @@
+<div class="title-box" data-class="Code">
+    <div class="title-box-left-items">
+        <svg class="title-box-svg" viewBox="0 0 1024 1024"
+             xmlns="http://www.w3.org/2000/svg">
+            <path d="M258.56 869.546667a106.666667 106.666667 0 0 1-75.093333-180.906667l288.426666-288.213333A202.24 202.24 0 0 1 768 177.066667a21.333333 21.333333 0 0 1 10.666667 15.573333 21.333333 21.333333 0 0 1-6.186667 18.133333l-99.84 99.84 38.186667 38.186667L810.666667 248.96a21.333333 21.333333 0 0 1 33.493333 4.48 203.093333 203.093333 0 0 1-32.853333 243.2 202.666667 202.666667 0 0 1-190.506667 53.76L333.44 838.613333a105.386667 105.386667 0 0 1-74.88 30.933334zM668.8 193.493333a159.786667 159.786667 0 0 0-152.533333 207.786667 21.333333 21.333333 0 0 1-6.826667 23.04l-2.773333 2.133333-2.346667 1.706667L213.333333 718.933333a64 64 0 0 0 0 89.386667 64 64 0 0 0 89.386667 0l291.2-290.56a9.386667 9.386667 0 0 0 1.706667-2.346667L597.333333 512a21.333333 21.333333 0 0 1 23.04-7.04 159.786667 159.786667 0 0 0 199.04-205.013333l-93.226666 93.226666a21.333333 21.333333 0 0 1-30.08 0l-68.48-68.48a21.333333 21.333333 0 0 1-6.186667-14.933333 21.333333 21.333333 0 0 1 6.186667-15.146667l93.226666-93.226666a158.506667 158.506667 0 0 0-52.053333-7.893334z"></path>
+            <path d="M284.8 760.533333a21.333333 21.333333 0 0 1-15.146667-36.266666l181.76-181.76a21.333333 21.333333 0 0 1 30.293334 0 21.333333 21.333333 0 0 1 0 30.08l-181.76 181.76a21.333333 21.333333 0 0 1-15.146667 6.186666z"></path>
+        </svg>
+        <span>Code</span>
```

Comment:
When the mouse clicks inside the code entry box, it drags the node instead of selecting the content